inc/class-upgrade.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * @package    WPSEO\Internal
+ */
+
+/**
+ * This code handles the option upgrades
+ */
+class WPSEO_Upgrade {
+
+	private $options = array();
+	
+	public function __construct() {
+		$this->options = WPSEO_Options::get_all();
+		
+		WPSEO_Options::maybe_set_multisite_defaults( false );
+
+		if ( version_compare( $this->options['version'], '1.5.0', '<' ) ) {
```

Comment:
Yes the WPSEO_Options class enforces that. It might be an empty string, but then this still returns true.